### PR TITLE
fix(#1717): Correct object name

### PR DESCRIPTION
**Pull request by volodya-lombrozo**
1. I've added `JavaPath` class that creates correct path for `java` object from `eo` object name. That class will be reused in `PhPackage`
2. Tests were added for `JavaPath`

Closes: #1717

### CODE REVIEW

**Comment by volodya-lombrozo** [comment]
@mximp Can you take a look, please?

File: eo-runtime/src/main/java/org/eolang/JavaPath.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.eolang;
+
+/**
+ * Java path.
```
Comment (mximp) [inline eo-runtime/src/main/java/org/eolang/JavaPath.java:27]: @volodya-lombrozo Please extend the description. What is this class for? Which Java path is meant?

File: eo-runtime/src/test/java/org/eolang/JavaPathTest.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+/**
+ * Test case for {@link JavaPath}.
+ *
+ * @since 0.29
+ */
+class JavaPathTest {
+
+    @ParameterizedTest
+    @MethodSource("cases")
```
Comment (mximp) [inline eo-runtime/src/test/java/org/eolang/JavaPathTest.java:41]: @volodya-lombrozo I believe `CsvSource` would be more readable and less verbose here. No need in additional method which is basically just a constant.

Comment (volodya-lombrozo) [reply eo-runtime/src/test/java/org/eolang/JavaPathTest.java]: @mximp Good suggestion. Thank you!

File: eo-runtime/src/main/java/org/eolang/JavaPath.java (hunk 1 above)
Comment (volodya-lombrozo) [reply eo-runtime/src/main/java/org/eolang/JavaPath.java:27]: @mximp I've extended `JavaPath` description. If it isn't clear, let me know.

**Comment by volodya-lombrozo** [comment]
@mximp Done. Can you have a look again, please?

**Review by mximp — APPROVED** [review]

**Comment by volodya-lombrozo** [comment]
@yegor256 Can you merge that changes, please?

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**